modules/terraform/format.go

```diff
@@ -14,9 +14,8 @@ import (
 var TerraformCommandsWithLockSupport = []string{
 	"plan",
 	"apply",
-	"apply-all",
+	"run-all",
 	"destroy",
-	"destroy-all",
```

Comment:
For backwards compatibility, we may need to keep these?

Reply:
ok I have added back those...